_templates/layout.html

```diff
@@ -31,35 +31,35 @@
   <div class="admonition-grid {% if godot_is_latest and godot_show_article_status %}admonition-grid-2x{% endif %}">
```

Comment:
```suggestion
  <div class="admonition-grid">
```

Also don't need that anymore, as we will only have one note at all times.